test/semantics/getdefinition01.f90

```diff
@@ -0,0 +1,51 @@
+! Copyright (c) 2018-2019, NVIDIA CORPORATION.  All rights reserved.
+!
+! Licensed under the Apache License, Version 2.0 (the "License");
+! you may not use this file except in compliance with the License.
+! You may obtain a copy of the License at
+!
+!     http://www.apache.org/licenses/LICENSE-2.0
+!
+! Unless required by applicable law or agreed to in writing, software
+! distributed under the License is distributed on an "AS IS" BASIS,
+! WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+! See the License for the specific language governing permissions and
+! limitations under the License.
+
+! Tests -fget-definition returning source position of symbol definition.
+
+!DEF: /m Module
```

Comment:
What's the purpose of the "DEF:" and "REF:" comments.

Reply:
These comments were copied over from another test. They act as sanity check while I was writing the checks, and they also act as sporadic comments for testing purposes.